README.md

```diff
@@ -55,13 +55,13 @@ This will output the translatable text to an POT file.
 After the text has been translated, create a new translated schema with:
 
 ```
-pipenv run python -m eq_translations.cli.translate_schema <schema_file> <translation_path> <output_directory>
+pipenv run python -m eq_translations.cli.translate_schema <schema_file> <translation_path> <output_directory> <target_language_code>
```

Comment:
The target language code should be read from the po file. Currently you can do something like:

```
pipenv run python -m eq_translations.cli.translate_schema tests/schemas/en/test_language.json tests/schemas/test_language-cy.po output/ eo
```

And it completes successfully.

Reply:
Language code is now read from the po file.